src/app/page.tsx

```diff
@@ -1,113 +1,48 @@
-import Image from 'next/image'
+'use client'
+
+import React, { useState, ChangeEvent, useRef } from 'react';
 
 export default function Home() {
+    const [images, setImages] = useState<File[]>([]);
+    const fileInputRef = useRef<HTMLInputElement | null>(null);
+  
+    const handleImageSelect = (event: ChangeEvent<HTMLInputElement>) => {
+      const selectedImages = Array.from(event.target.files as FileList);
+      setImages(images.concat(selectedImages));
+    };
+  
+    const handleImageDelete = (index: number) => {
+      const updatedImages = images.slice();
+      updatedImages.splice(index, 1);
+      setImages(updatedImages);
+    };
+
+    const handleClick = () => {
+      if (fileInputRef.current) {
+        fileInputRef.current.click();
+      }
+  }
   return (
     <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex">
-        <p className="fixed left-0 top-0 flex w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">
-          Get started by editing&nbsp;
-          <code className="font-mono font-bold">src/app/page.tsx</code>
-        </p>
-        <div className="fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none">
-          <a
-            className="pointer-events-none flex place-items-center gap-2 p-8 lg:pointer-events-auto lg:p-0"
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{' '}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className="dark:invert"
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
+      <h1>HDRICalibrationTool</h1>
+      <div>
+        <div className="image-preview" style={{ display: 'flex', flexWrap: 'wrap'}}>
+          {images.map((image, index) => (
+            <div key={index} className="image-item">
+              <div className="grid grid-cols-3 gap-4"> {/* option 2*/}
```

Comment:
Using a `grid` here makes the mage, button and name take 3 columns, which take up lots of space for no reason. I was thinking of either using the grid instead of the flex (above) or define this component as `inline-block` instead.